modules/core/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ ocv_add_dispatched_file(has_non_zero SSE2 AVX2 LASX )
 ocv_add_dispatched_file(matmul SSE2 SSE4_1 AVX2 AVX512_SKX NEON_DOTPROD LASX)
 ocv_add_dispatched_file(mean SSE2 AVX2 LASX)
 ocv_add_dispatched_file(merge SSE2 AVX2 LASX)
+ocv_add_dispatched_file(minmax SSE2 SSE4_1 AVX2 VSX3 LASX)
```

Comment:
Such kind of optimizations should be done on 4.x branch first according to existed policy: https://github.com/opencv/opencv/wiki/Branches